### PR TITLE
[hotfix] remove unused import from tiledacq that caused fastem-calibr…

### PR DESCRIPTION
**Pull request by tepals**
…ations CI to fail

### CODE REVIEW

**Review by nandishjpatel — APPROVED** [review]

File: src/odemis/acq/stitching/_tiledacq.py [hunk 1]
```diff
@@ -34,7 +34,6 @@
 from odemis.acq.stream import Stream, EMStream, ARStream, \
     SpectrumStream, FluoStream, MultipleDetectorStream, util, executeAsyncTask, \
     CLStream
-from odemis.gui import conf
 from odemis.gui.util.raster import point_in_polygon
```
Comment (pieleric) [inline src/odemis/acq/stitching/_tiledacq.py:37]: Actually, that import is also a bit fishy: normally the .acq should not have any import from .gui . If we need this function, we probably could move gui.util.raster to util.raster.

**Review by pieleric — APPROVED** [review]

**Comment by pieleric** [comment]
Could you explain in the commit log how an import can cause the CI to fail?

File: src/odemis/acq/stitching/_tiledacq.py (hunk 1 above)
Comment (tepals) [reply src/odemis/acq/stitching/_tiledacq.py:37]: good point. I don't really see a reason why `raster.py` is in `gui.util` and it is a very useful function to have, so I will move `gui.util.raster` to `util.raster`

Comment (tepals) [reply src/odemis/acq/stitching/_tiledacq.py:37]: should I do this in a separate PR?

Comment (pieleric) [reply src/odemis/acq/stitching/_tiledacq.py:37]: Yes, that's probably the best.

**Pull request merged**